airflow/contrib/operators/ssh_execute_operator.py

```diff
@@ -55,7 +55,7 @@ def __enter__(self):
         prefix = self._prefix
 
         pmktemp = ssh_hook.Popen(["-q",
-                                  "mktemp", "--tmpdir", prefix + "_XXXXXX"],
+                                  "mktemp", "-t", prefix + "_XXXXXX"],
```

Comment:
@zodiac oops, yes absolutely. I thought we were passing arguments. Ok then I am not particularly against this change, although the better approach (due to -t being deprecated, although i dont see it going away soon) maybe just skip the test on  MacOS as it is not a deployment platform?